docs/antora-playbook-local.yml

```diff
@@ -1,7 +1,6 @@
 site:
   title: "Akka Documentation"
   url: https://doc.akka.io
-  start_page: ROOT::index.adoc
```

Comment:
Removes this warn: `WARN (@antora/content-classifier): Start page specified for site not found: ROOT::index.adoc`

start_page [defaults](https://docs.antora.org/antora/latest/component-start-page/#default-start-page) to index.adoc from root anyway.